src/io/perun.clj

```diff
@@ -594,23 +661,19 @@
   on the provided `fileset` and `options`."
   [task-name fileset options]
   (let [global-meta (pm/get-global-meta fileset)
-        grouper (:grouper options)
-        paths (->> fileset
-                   pm/get-meta
-                   (filter (:filterer options))
-                   grouper)]
+        {:keys [grouper sortby comparator out-dir]} options
```

Comment:
This should be in the method signature like so `[task-name fileset {:keys [grouper sortby comparator out-dir] :as options}]'